.github/workflows/ci.yml

```diff
@@ -11,44 +11,68 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - name: Set up Go
+      -
+        name: Checkout
+        uses: actions/checkout@v2.3.4
```

Comment:
Actions which rely on Node 12 are deprecated. For any of the GitHub made ones we can safely upgrade to @v3.

> Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout@v2.3.4, actions/setup-go@v2, docker/metadata-action@v3, docker/setup-qemu-action@v1, docker/setup-buildx-action@v1, docker/build-push-action@v2

Reply:
All done

Reply:
Still getting a warning for using @v2 of the checkout action: https://github.com/EventStore/es-gencert-cli/actions/runs/3547339752

Reply:
Yeah, but 2.5.0 is the latest release https://github.com/actions/checkout/releases

Reply:
Ok, I see that 3.1 is there.

Reply:
No more warnings